src/main/resources/application.properties

```
@@ -1 +1,8 @@
 spring.application.name=bookApp
+spring.datasource.url=jdbc:mysql://localhost:3306/cinemaNew?serverTimeZone=UTC
```

for the future - SQL dont cares about registry case, so while naming, better to prefer snake_case over camelCase
```suggestion
spring.datasource.url=jdbc:mysql://localhost:3306/cinema_new?serverTimeZone=UTC
```

---

Not fixed